content/_data/changelogs/weekly.yml

```diff
@@ -6293,6 +6293,189 @@
   # pull: 4494 (PR title: Remove references to Azure maven cache)
   # pull: 4502 (PR title: [JENKINS-61102] Use XMLUnit to fix test)
 
+- version: '2.222'
+  date: 2020-02-23
+  changes:
+  - type: major rfe
+    category: major rfe
+    pull: 4463
+    issue: 60920
+    authors:
+    - fqueiruga
+    - daniel-beck
+    message: |-
+      Revamp the layout and icons of the header bar and breadcrumbs.
+      Instances with plugins that depend on details of the Jenkins layout (e.g. Simple Theme Plugin) may experience UI/layout problems.
+      A new experimental header color scheme can be enabled by setting the <code>jenkins.ui.refresh</code> system property to <code>true</code>.
+  - type: major rfe
+    category: major rfe
+    pull: 4463
+    authors:
+    - fqueiruga
+    - daniel-beck
+    message: |-
+      Introduce a new experimental UI that can be enabled by setting the <code>jenkins.ui.refresh</code> system property to <code>true</code>.
+      Currently it includes a new header color scheme, more changes to be added as a part of the UI/UX revamp.
+    references:
+      - pull: 4463
+      - issue: 60920
+      - url: https://github.com/jenkinsci/jep/blob/master/jep/223/README.adoc
+        title: JEP-223
+      - url: https://jenkins.io/sigs/ux/
+        title: Jenkins UX SIG
+  - type: major rfe
+    category: major rfe
+    pull: 4501
+    authors:
+    - daniel-beck
+    - timja
+    references:
+      - pull: 4501
+      - issue: 60266
+      - url: https://github.com/jenkinsci/jep/blob/master/jep/223/README.adoc
+        title: JEP-223
+    message: |-
+      Add a new experimental <code>Overall/Manage</code> permission which allows a user to configure parts of the global Jenkins configuration without having the <code>Overall/Administer</code> permission.
+      This is an experimental feature, disabled by default, that can be enabled by setting the <code>jenkins.security.ManagePermission</code> system property to <code>true</code>.
+  - type: major rfe
+    category: major rfe
+    pull: 4506
+    authors:
+    - daniel-beck
+    - timja
+    references:
+      - pull: 4506
+      - issue: 12548
+      - url: https://github.com/jenkinsci/jep/blob/master/jep/224/README.adoc
+        title: JEP-224
+      - url: https://plugins.jenkins.io/extended-read-permission/
+        title: Extended Read Permission plugin
+    message: |-
+      Add a new experimental `Overall/SystemRead` permission, which gives (almost) full read access to the Jenkins instance.
+      The permission is disabled by default, install the Extended Read Permission plugin plugin to activate it.
```

Comment:
```suggestion
      The permission is disabled by default, install the Extended Read Permission plugin to activate it.
```

Reply:
(still need to get extended read permission plugin shipped with system read but probably fine for this to go in like this)